Automattic/newspack-plugin
# feat: ads marketplace orders ui

### All Submissions:






### Changes proposed in this Pull Request:

1203270678582143-as-1204470726175919

Implements a sample dashboard for Ads Marketplace with ad order listing and management.

<img width="1529" alt="image" src="https://github.com/Automattic/newspack-plugin/assets/820752/2de4ef6d-813c-4a6d-88f0-dd3ce94dabd9">

The "Settings" is a placeholder that will later allow:
 - Triggering email when an ad order is placed
 - Which email address should receive the email

The "Overview" section is a placeholder/sample for later integration with GAM data.

### How to test the changes in this Pull Request:

1. Check out this branch and https://github.com/Automattic/newspack-ads/pull/694
2. Visit Newspack -> Advertising -> Marketplace and confirm it renders a dashboard as in the image above
3. Click "Products" and confirm you can see and create products
4. Make sure you have valid Ad products
5. Add the "Marketplace" block on a page, visit and place an ad order
6. Confirm the order is placed and it's visible on the dashboard
7. Click on the link to visit the order in GAM and approve the order
8. Back to the dashboard, click on "Fetch GAM order status"
9. Confirm the order shows as approved
10. Click "Orders" and confirm all ad-related orders appear

### Other information:
